src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -234,6 +236,7 @@
   <Target Name="CreateVersionFileDuringBuild" Condition="'$(SkipVersionGeneration)'!='true' AND '$(ShouldCreateVersionFileDuringBuild)'=='true'" DependsOnTargets="CreateOrUpdateCurrentVersionFile">
     <PropertyGroup>
       <BuildNumberMajor>$(GeneratedBuildNumberMajor)</BuildNumberMajor>
```

Comment:
No, the assignments we do here are inside a target, and that target will get skipped if SkipVersionGeneration is true